backstage-templates-for-eks/s3-bucket/template-s3-bucket.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  description: Creates an s3 bucket
+  name: s3-bucket
+  title: Creates an Amazon S3 Bucket
+spec:
+  owner: guest
+  type: service
+  parameters:
+    - properties:
+        tfVars:
+          properties:
+            aws_region:
+              description: AWS Region
+              type: string
+          required:
+            - aws_region
+          title: Terraform variables
+          type: object
+      title: Terraform config options
+    - properties:
+        adminRoleName:
+          default: Admin
+          description: Name of the role to give the administrative rights.
+          title: Admin Role Name
+          type: string
+        name:
+          title: name of this s3 bucket
+          type: string
+        namespace:
+          default: flux-system
+          title: Existing namespace to create this resource
+          type: string
+      title: Configuration Options
```

Comment:
can we make it all part of one configuration rather than two sets of configurations? looks small enough

Reply:
fixed.